app/Resources/views/adherent/_membership_form.html.twig

```diff
@@ -123,6 +123,9 @@
 
     <input type="hidden" class="nojs-g-recaptcha-response" name="g-recaptcha-response" value="" />
     {% if recaptcha_public_key is not empty %}
+        {% for flashMessage in app.session.flashbag.get("error_recaptcha") %}
```

Comment:
You can leverage the new 3.3 feature here:
`app.session.flashbag.get("error_recaptcha")` => `app.flashes("error_recaptcha")`